cmd/collectors.go

```diff
@@ -56,8 +56,8 @@ func parseCollector(s string) (t, arg string) {
 }
 
 func newCollector(t, arg string, src *lib.SourceData, conf Config) (lib.Collector, error) {
-	loadConfig := func(out encoding.TextUnmarshaler) error {
-		if err := envconfig.Process("k6", out); err != nil {
+	loadConfig := func(out encoding.TextUnmarshaler, namespace string) error {
```

Comment:
I added an extra arg to loadConfig func. What are the use cases for K6 env var?

Reply:
What do you mean?

Reply:
I've extracted the hard-coded **k6** string and made it a var that is set when the config is loaded. The question was to understand what are the use cases for k6 usage and if we would need to provide a k6+statsd / k6+datadog combination.

Reply:
Ah, what I answered in my other comment?

Reply:
```
Possible workaround:

// package shared
type Config {
  Addr string `envconfig:"ADDR"`
}

// package statsd
type Config struct {
  shared.Config `envconfig:"STATSD"`
}

// package dogstatsd
type Config struct {
  shared.Config `envconfig:"DOGSTATSD"`
}
```

and 

```
You could also do something like just setting envconfig:"ADDR" and then using a different prefix in envconfig.Process().
```

Reply:
I'm sorry, I'm not sure what the question is here... ^^;;

Reply:
^^:: No worries. Probably I'm just overcomplicating things. Are you happy with this change?

Reply:
Definitely!

Reply:
Honestly, at this point you could just change the `loadConfig` calls to `envconfig.Process` calls wholesale. That function basically existed to not have to type (and potentially typo "k6") all the time.

Reply:
I agree but I tried to keep it consistent to how was done before. Happy to do other PR to simplify loadConfig stuff.